src/frontend/src/handler/create_table.rs

```diff
@@ -188,10 +188,11 @@ pub fn bind_sql_table_constraints(
     let mut columns_catalog = column_descs
         .into_iter()
         .map(|c| {
+            // All columns except `_row_id` or starts with `_rw` should be visible.
+            let is_hidden = c.name.starts_with("_rw");
```

Comment:
> All columns except `_row_id` or starts with `_rw` should be visible.

is it a convention?

Reply:
No, but currently we only have this hidden column, so I just want to leave it here.